### ISSUE TITLE
Add a linter

### ISSUE THREAD

**Issue by EricPickup** [UWindsorCSS/hub#143]
Rubocop maybe?

**Comment by amanpatel123** [comment]
Yeah, we use Rubocop at Fullscript and I love it. We should go with it!

**Comment by amanpatel123** [comment]
I came across this GitHub Action: [Rubocop-linter](https://github.com/andrewmcodes/rubocop-linter-action). Although it has less stars but the code coverage is excellent and test's are passing. Should we use this @EricPickup? Its better than creating our own.

**Comment by EricPickup** [comment]
let's try it out!
